packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -469,9 +471,12 @@ export default {
                     address: this.address,
                     customer: this.customer,
                     isCheckoutMethodDelivery: this.isCheckoutMethodDelivery,
-                    time: this.time,
+                    time: this.hasAsapSelected
+                        ? this.availableFulfilment.times[0]
+                        : this.time,
```

Comment:
Might be worth pushing this logic back into the store, because as the store is aware of the `availableFulfilment` values, it can update `time` as part of the commit when ASAP is set to true. 

Unless theres some use in retaining the previous value of `time`?

Reply:
Fixed.